src/agentlab/experiments/study.py

```diff
@@ -220,6 +226,9 @@ def name(self):
             study_name = f"{agent_names[0]}_on_{self.benchmark.name}"
         else:
             study_name = f"{len(agent_names)}_agents_on_{self.benchmark.name}"
+
+        study_name = slugify(study_name, max_length=50, allow_unicode=True)
```

Comment:
was there study_names that were problematics? or it's just in case?

Should we put more than 50 as max length?

Reply:
Done, I increased to 100